src/actions-on-google.ts

```diff
@@ -675,8 +686,15 @@ export class ActionsOnGoogle {
                         }
                     }
                 }
+                if (data.screen_out && data.screen_out.format === 'HTML' && data.screen_out.data) {
+                    assistResponse.screenOutHtml = data.screen_out.data.toString()
```

Comment:
This is just going to return a full HTML page? Is that going to be helpful?

Reply:
For our purposes, yes. With Botium we are developing automated training and testing for bots and voice assistants. we try to make test results as descriptive as possible, and showing the screen output to the user is clearly helpful when writing test cases.

Reply:
Will make this an option.